habitat-lab/habitat/tasks/rearrange/social_nav/social_nav_task.py

```diff
@@ -121,9 +120,7 @@ def reset(self, episode: Episode):
 
         super().reset(episode)
 
-        self.pddl_problem.bind_to_instance(
```

Comment:
um are we sure no other tasks are using this?

Reply:
Yes, and the tests would also catch it if they were.